test/JsBuiltIns/ChakraLib.js

```diff
@@ -11,6 +11,8 @@ var tests = [
         body: function () {
             assert.isNotUndefined(__chakraLibrary, "__chakraLibrary must NOT be undefined as we're using the -LdChakraLib flag.");
         },
+    },
+    {

```

Comment:
Was this preventing the above test from running at all or?

Reply:
It was only running one test, I am not sure if it was the first or second, because it was basically one object where name and body were defined twice. Not sure which took precedence. I was only looking to see how JsBuiltIns got tested, and the LdChakraLib flag/opcode seemed a bit too over the top for our needs.